src/probnum/prob/distributions/normal.py

```diff
@@ -468,7 +472,7 @@ def pdf(self, x):
                                                            mean=self.mean().ravel(),
                                                            cov=self.cov())
         # TODO: this reshape is incorrect, write test for multiple matrices
```

Comment:
what about this?

Reply:
This will be eventually resolved by work from Aljoscha. Removed Todos and created an issue instead.